Added information, reformated to make a little more sense (#1)
iPherian/Documentation: _csharp-api/inputsystem/input.md

        
    } 
  ```

>  #### [Notice that one could document the stages of a key using the above 3](#Monitoring-a-key's-state)

Reviewer: I think the link could be fixed like so. Also thought the link text could be shortened? I find long linkable text to be kind've ugly.

Btw to get the syntax of a link to another section, view the file in your [repo](https://github.com/TakingTheRisk/Documentation/blob/patch-2/_csharp-api/inputsystem/input.md) and mouse over to the left of the place you want to link to and you'll see a chain icon which has (at the end of it) the correct # prefixed link.

```suggestion
 #### To see how one could document the stages of a key using the above 3 methods: [example](#monitoring-a-keys-state)
```
